org.musicbrainz.Picard.appdata.xml

```diff
@@ -0,0 +1,73 @@
+
+  <description>
+    <p>
+      Do you need to clean up your music library? Picard is an open-source
+      cross-platform music tagger by MusicBrainz. It has the ability to
+      identify audio files without any metadata.
+    </p>
+    <p>
+      A variety of plugins are available and you can even write your own.
+      Picard supports a wide range of audio formats and can also lookup an
+      entire CD for you.
+    </p>
+    <p>
+      Tagging audio files has never been easier.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">org.musicbrainz.Picard.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">https://www.daitauha.fr/static/screenshots/picard/main.png</screenshot>
+  </screenshots>
+
+  <url type="homepage">https://picard.musicbrainz.org/</url>
+  <url type="bugtracker">https://tickets.metabrainz.org/projects/PICARD/issues/</url>
+  <url type="faq">https://picard.musicbrainz.org/docs/faq/</url>
+  <url type="help">https://community.metabrainz.org/c/picard</help>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="violence-cartoon">none</content_attribute>
+    <content_attribute id="violence-fantasy">none</content_attribute>
+    <content_attribute id="violence-realistic">none</content_attribute>
+    <content_attribute id="violence-bloodshed">none</content_attribute>
+    <content_attribute id="violence-sexual">none</content_attribute>
+    <content_attribute id="violence-desecration">none</content_attribute>
+    <content_attribute id="violence-slavery">none</content_attribute>
+    <content_attribute id="violence-worship">none</content_attribute>
+    <content_attribute id="drugs-alcohol">none</content_attribute>
+    <content_attribute id="drugs-narcotics">none</content_attribute>
+    <content_attribute id="drugs-tobacco">none</content_attribute>
+    <content_attribute id="sex-nudity">none</content_attribute>
+    <content_attribute id="sex-themes">none</content_attribute>
+    <content_attribute id="sex-homosexuality">none</content_attribute>
+    <content_attribute id="sex-prostitution">none</content_attribute>
+    <content_attribute id="sex-adultery">none</content_attribute>
+    <content_attribute id="sex-appearance">none</content_attribute>
+    <content_attribute id="language-profanity">none</content_attribute>
+    <content_attribute id="language-humor">none</content_attribute>
+    <content_attribute id="language-discrimination">none</content_attribute>
+    <content_attribute id="social-chat">none</content_attribute>
+    <content_attribute id="social-info">none</content_attribute>
+    <content_attribute id="social-audio">none</content_attribute>
+    <content_attribute id="social-location">none</content_attribute>
+    <content_attribute id="social-contacts">none</content_attribute>
+    <content_attribute id="money-purchasing">none</content_attribute>
+    <content_attribute id="money-gambling">none</content_attribute>
```

Comment:
AFAICT, [according to the spec](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-content_rating), you only need to specify the ones that are *not* "none".

Reply:
I did not alter that because it was present in the base file and also in some appdata files I saw in other projects. Should I remove the unneeded ones?

Reply:
Not sure really, is it ok to remove all of them if they are all none? @bochecha, since you added this initially any thoughts on it? I am not too familiar with OARS data inside appstream.

Reply:
If you remove all of them, then you end up not having any OARS metadata at all, which software centers interpret as "not rated" rather than "rated safe".

Reply:
@bochecha That's contrary to the spec though. :/ It specifically says:
> In case the `<content_rating/>` tag is empty (no `<content_attribute/>` is present), it is assumed that the component was checked for age ratings and no age restrictions apply.

However, it that's the de facto standard, then that's how it is.

Reply:
@Freso @bochecha Since now we need to set content attribute money-purchasing to _mild_. I guess we could omit the unneeded ones.

Reply:
> In case the <content_rating/> tag is empty

Oh, maybe I'm mixing up between an empty `<content_rating/>` and no `<content_rating/>` at all?

I'll try an empty `<content_rating/>` in the Flathub build, and see what GNOME Software has to say about it. :slightly_smiling_face:

Reply:
We can do that once it is resolved whether to use _money-purchasing_ to _none_ or _mild_ .

Reply:
Agreed with @abhi-ohri . So let's set money-purchasing to mild and remove everything else.

Reply:
SInce the `content_rating` now has a content-attribute `money-purchasing` we can remove the unneeded tags. @Freso What's your say?